axlearn/cloud/gcp/measurement.py

```diff
@@ -13,10 +16,14 @@
         --recorder_spec=name=my-run-with-goodput \
         --recorder_spec=upload_dir=my-output-directory/summaries \
         --recorder_spec=upload_interval=30 \
-        --recorder_spec=step_deviation_interval_seconds=30
+        --recorder_spec=rolling_window_size=86400,259200,432000
```

Comment:
We decide to use last 1/3/7 rolling window.

In the examples, can we change to 
```
86400,259200,604800
```
? Tgabjs